raster-foundry/raster-foundry
# Move scene thumnbnails to tile server

## Overview

This PR moves scene thumbnails to the tile server and makes all previews and thumbnail coloring consistent.

### Checklist


### Notes

I uncovered ~(or possibly created)~ a new bug where the scene list doesn't get filled in correctly so this might be WIP still but at least colors finally work. ETA: I didn't create it! Tracked as #4652 

## Testing Instructions

 * filter for some ingested scenes
 * look at their thumbnails
 * hover over them to look at them on the map
 * click on them to look at them in the scene detail modal
 * all the colors should match

Closes #4468 

Closes #4633
